### DISCUSSION TITLE
Trigger notifications for specific match files

### DISCUSSION THREAD

**Discussion by greggathy** [espanso/espanso#1478] [Feature requests and Ideas]
Love espanso and can't live without it now!
I've just started a new match file which I'll populate over time with my common typos and misspellings to help me out in apps that don't have built in spellchecks. 
I normally have all espanso notifications disabled because I always know it's working when I invoke the trigger, but it would be cool to know that it's made a replacement of a word that I misspelled (matchs in that specific spellchecker.yml) that I may have done accidentally without realising. 
To implement this I envision a type of "show_notifications: true" that may optionally be set at the top of match files and only applies to matches made in that file's triggers.

**Comment by smeech** [comment]
Sorry about the late reply - I am trying to mop up and sort the old Issues and Discussions.

Nice idea. Others (#1170) have suggested a “bell” sound when an expansion is triggered. I'm just happy my spelling's been corrected!

**Comment by AucaCoyan** [comment]
Created https://github.com/espanso/espanso/issues/2135 to track it
